assets/javascripts/discourse/routes/admin-plugins-show-discourse-ai-ai-personas.js

```diff
@@ -1,7 +1,7 @@
 import DiscourseRoute from "discourse/routes/discourse";
 
-export default DiscourseRoute.extend({
-  async model() {
+export default class DiscourseAiAiPersonasRoute extends DiscourseRoute {
```

Comment:
It seems like `AiAi` is intentional here from concatenating `DiscourseAi` and `AiPersonasRoute`, but it sure looks like a typo. 😅 

I guess it's a question of whether "persona" needs to have "AI" prefixed in the context of the `discourse-ai` plugin, but that's out of scope for this PR.